client/src/clusterStatus/localStatus.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from "react";
+import { useEffect, useState } from "react";
+import { List, Modal } from "semantic-ui-react";
+import { HostDetails } from "../messages";
+
+interface LocalStatusProps {
+    localCore: number;
+    cudas: number[];
+    details: HostDetails[];
+}
+
+const LocalStatus: React.FC<LocalStatusProps> = ({ localCore, cudas, details }) => {
+    const [cudaText, setcudaText] = useState("");
```

Comment:
Why does this need to be a state? Is it not possible to just use a local variable that is set depending on the values of `cudas`?

Reply:
I will check that out :+1: